package.json

```diff
@@ -58,5 +58,12 @@
     "library"
   ],
   "license": "MIT",
-  "repository": "github:creativecommons/fonts"
+  "repository": "github:creativecommons/fonts",
+  "dependencies": {
+    "jest": "^25.2.7",
+    "jest-image-snapshot": "^3.0.1",
+    "jest-puppeteer": "^4.4.0",
+    "puppeteer": "^2.1.1",
+    "start-server-and-test": "^1.10.11"
```

Comment:
These already exist as `devDependencies`. Remove from here.

Reply:
Okay @dhruvkb will update the following